github-dark.css

```diff
@@ -942,7 +942,8 @@
   .ajax-pagination-form .ajax-pagination-btn:hover,
   .ajax-pagination-form .ajax-pagination-btn:focus,
   .manage-repo-access-not-active, .header-search-scope, .reverse-progress-bar,
-  .featurette.py-5, .ace_button:hover {
+  .featurette.py-5, .ace_button:hover,
+  .payment-history .refunded, .payment-history .failed, {
```

Comment:
I tested it again on the cusom code font line and found out that the error is because I ran perfectionist before lint.

Before perfectionist (no lint error)

```css
, .blob-code-inner, {
```

After Perfectionist

```css
, .blob-code-inner,  {
```

perfectionist adds an extra space, so the linter threw this error:

> github-dark.css
 19:34  ×  Expected single space before "{"   block-opening-brace-space-before

>\>\> Exited with code: 1.
Warning: Task "exec:stylelint" failed. Use --force to continue.

> Aborted due to warnings.